dev-requirements-2.9-py2.txt

```diff
@@ -0,0 +1,12 @@
+behave==1.2.6
```

Comment:
is py2 still needed?

Reply:
Likely not required. The fact that support is broken is a bug in tableschema, however.